memory/player_party_manager.py

```diff
@@ -58,42 +59,36 @@ def update(self: Self) -> None:
                 # logger.debug(f"PlayerPartyManager Reloading {type(_e)}")
                 self.__init__()
 
-    def _read_current_party(self: Self) -> [PlayerPartyCharacter]:
+    def _read_current_party(self: Self) -> None:
         try:
```

Comment:
Potentially this try should be for the entire function? Can the call `combat_players = self.memory.follow_pointer(...` fail?

Reply:
Also, completely missed the weird return value :P

Reply:
All memory can but not likely if the main one fails. This one would be a nice crash to know about if it does